app/src/main/java/org/mozilla/fenix/components/BackgroundServices.kt

```diff
@@ -61,12 +62,20 @@ class BackgroundServices(
         }
     }
 
-    fun defaultDeviceName(context: Context): String = context.getString(
-        R.string.default_device_name,
-        context.getString(R.string.app_name),
-        Build.MANUFACTURER,
-        Build.MODEL
-    )
+    fun defaultDeviceName(context: Context): String = try {
+            context.getString(
+                R.string.default_device_name,
+                context.getString(R.string.app_name),
+                Build.MANUFACTURER,
+                Build.MODEL
+            )
+        } catch (ex: FormatFlagsConversionMismatchException) {
```

Comment:
Nit: since this is a temporary fix, would you mind adding a TODO pointing to the issue and with a quick explanation?  Something small, like `// A malformed string is causing crashes. This will be removed when the string is fixed. See #5552`